src/Gaffer/Context.cpp

```diff
@@ -477,3 +480,17 @@ const std::string &Context::SubstitutionProvider::variable( const boost::string_
 	m_formattedString.clear();
 	return m_formattedString;
 }
+
+void Context::validateHashes()
+{
+	for( Map::iterator it = m_map.begin(), eIt = m_map.end(); it != eIt; ++it )
+	{
+		IECore::MurmurHash correct = it->second.data->Object::hash();
+		correct.append( (uint64_t)&it->first.string() );
```

Comment:
This is repeated in three spots - might be worth putting in a new `Storage::updateHash( name )` method?

Reply:
Sure - in a way it would be nice if it was a private method on Context, so it could set `m_hashValid = false` as well, but it's probably cleaner to put it on Storage and pair it with a `m_hashValid = false`

Reply:
Fixed in 7e95c0f